src/app/participant-page/participant-page.component.html

```diff
@@ -136,7 +136,13 @@ <h3 class="Color--warn Line--Break">Lost to Follow-Up</h3>
           </tr>
           <tr>
             <td>Do Not Contact</td>
-            <td>{{getUtil().getYesNo(participant.data.profile['doNotContact'])}}</td>
+            <td *ngIf="hasRole().allowedToEditParticipant()">
+              <md-checkbox [value]="updatedDNC" color="primary" disableRipple
```

Comment:
I tried [ngModel] [(ngModel)] [checked]
at the end it is always selected even if updatedDNC is false per debugger

Reply:
I believe it should be 
```
<md-checkbox [(ngModel)]="updatedDNC" color="primary" disableRipple
             (change)="updateDNC($event)">
</md-checkbox>
```